ui/apps/platform/src/routePaths.ts

```diff
@@ -291,15 +294,15 @@ const routeRequirementsMap: Record<RouteKey, RouteRequirements> = {
         ]),
     },
     'vulnerability-management': {
+        // Same resources as workload-cves although lack of commented-out resources affects entire list or entity pages.
         resourceAccessRequirements: everyResource([
-            'Alert', // for Cluster and Deployment and Namespace
-            'Cluster',
+            // 'Alert', // for Cluster and Deployment and Namespace
+            // 'Cluster',
             'Deployment',
             'Image',
-            'Namespace',
-            'Node',
+            // 'Namespace',
+            // 'Node',
             'WatchedImage', // for Image
```

Comment:
Although it's required, it really should not be in order to view images so I would recommend that we remove it as the errors are pretty clear that you need watchedimage rights. We will file a ticket to clean up this permission requirement

Reply:
Here are relevant screenshots that I had forgotten to take yesterday.

1. Visit /main/vulnerability-management with only Image and not Deployment resource.

    Keep Deployment resource as required because image CVEs are closely related to deployments, therefore data errors for widgets:
    * Top risky deployments by CVE count and CVSS score
    * Recently detected image vulnerabilities
    * Most common image vulnerabilities
        ![Deployment_Dashboard](https://github.com/stackrox/stackrox/assets/11862657/59bd8ed8-1c0c-466c-ace0-89847877ca6b)

2. Visit /main/vulnerability-management/images with Image and not WatchedImage resource.

    * Although data error prevents rendering the list, I commented out the resource as requested above.
        ![WatchedImage_Images](https://github.com/stackrox/stackrox/assets/11862657/c0314d68-5cf2-442c-b548-5bf17020a4ce)